variables.js

```diff
@@ -33,6 +42,9 @@
     2. Declare a variable that stores the age of a user. What name would you choose for this variable?
 *******************************************************************************/
 // TODO: ADD YOUR CODE BELOW
+let favoriteMovie = "lion";
+let userAge = "20";
```

Comment:
Good job for naming the variables, make sure the numeric values are assigned without enclosing them in double quotation marks.